operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -774,6 +778,29 @@ static CertificateFactory certificateFactory() {
         return factory;
     }
 
+    private void generateCaTrustStore(Subject subject, Map<String, String> certData) {
```

Comment:
Should this method be used also from `generateCaKeyAndCert` and `renewCaCert` to better share the code? They seem to have some stuff in common.

Reply:
Yes, I thought a lot about it, and I will continue on that but most is about moving useless `File` instances around in some cases. Btw I will think more.

Reply:
I managed to refactor and reusing the code in a couple of places. I have more ideas about cert generation and renewal places but not for this PR.